### PR TITLE
task: add missing feature flags for docs.rs on 0.2.x

### LINKED ISSUE #3206: Document that `task_local!` requires `rt-util` in tokio 0.2
**Version**
tokio 0.2.23

**Description**
Documentation of `tokio::task_local!` doesn't mention that it requires feature `rt-util`.

```
error[E0433]: failed to resolve: could not find `task_local` in `tokio`
   |
54 |     tokio::task_local! {
   |            ^^^^^^^^^^ could not find `task_local` in `tokio`
```

It should also be checked that 0.3 documentation for this macro is correct. It looks like `rt` feature might be required there which is not documented either. (But I didn't test this.)

### PR DESCRIPTION
**Pull request by nylonicious**
Fixes #3206 on `0.2.x` branch.

### CODE REVIEW

File: tokio/src/task/task_local.rs [hunk 1]
```diff
@@ -31,6 +31,7 @@ use std::{fmt, thread};
 ///
 /// [`tokio::task::LocalKey`]: struct@crate::task::LocalKey
 #[macro_export]
+#[cfg_attr(docsrs, doc(cfg(feature = "rt-util")))]
```
Comment (Darksonn) [inline tokio/src/task/task_local.rs:34]:
```suggestion
#[cfg_attr(docsrs, doc(cfg(all(feature = "rt-util", "rt-core"))))]
```

File: tokio/src/task/task_local.rs [hunk 2]
```diff
@@ -90,6 +91,7 @@ macro_rules! __task_local_inner {
 /// # }
 /// ```
 /// [`std::thread::LocalKey`]: struct@std::thread::LocalKey
+#[cfg_attr(docsrs, doc(cfg(feature = "rt-util")))]
```
Comment (Darksonn) [inline tokio/src/task/task_local.rs:94]:
```suggestion
#[cfg_attr(docsrs, doc(cfg(all(feature = "rt-util", "rt-core"))))]
```

**Review by Darksonn — CHANGES_REQUESTED** [review]
I made a quick test to double-check this, but they actually also require the `rt-core` feature.

**Comment by nylonicious** [comment]
Sorry, my bad I paid more attention to 0.3 version. Should be fine now I think.

**Review by Darksonn — APPROVED** [review]
Thanks!

**Pull request merged**